notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p><strong>If you don't already have a Cloud Storage bucket that you can use</strong>, run the following cell to create one. The bucket will be deleted in the&nbsp;<strong>Cleaning up</strong>&nbsp;section at the end of this tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>